src/main/java/ch/ivyteam/ivy/maven/InstallEngineMojo.java

```diff
@@ -62,7 +70,26 @@
   public static final String GOAL = "installEngine";
   public static final String ENGINE_LIST_URL_PROPERTY = "ivy.engine.list.url";
   public static final String DEFAULT_ARCH = "Slim_All_x64";
-  
+  public static final String ENGINE_ZIP_GAV_GROUP = "com.axonivy";
+  public static final String ENGINE_ZIP_GAV_ARTIFACT = "engine";
+
+  /**
+   * Indicate if the engine artifact should be downloaded using maven (from a configured maven repository) or if
+   * the URL download way should be used.
+   */
+  @Parameter(property="ivy.engine.download.from.maven", defaultValue = "false")
+  protected Boolean downloadUsingMaven;
+
+  /* maven artifact resolution components / params */
+  @Parameter(defaultValue = "${repositorySystemSession}", readonly = true)
+  private RepositorySystemSession repositorySession;
+
+  @Parameter(defaultValue = "${project.remotePluginRepositories}", readonly = true)
```

Comment:
streamline param names with AbstractProjectCompileMojo

Reply:
Hey, thanks for the review. What do you mean with stream line in that case? Shall I move the maven params (repo system, plugin repositories) to the AbstractProjectCompileMojo or is something wrong with the name?

Reply:
Forget about it :) 
i thought that the ArtifactRepository was injected with a different property name than it occures in the AbstractProjectCompileMojo. But now I see that you are reference pluginRepo, and not the artifact repo...